src/main/java/org/apache/maven/shared/dependency/analyzer/asm/ConstantPoolParser.java

```diff
@@ -21,6 +21,7 @@
 import java.nio.Buffer;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
+import java.text.ParseException;
```

Comment:
`ParseException` does not make much sense here imho.
There's a [`VisitClassException`](https://github.com/apache/maven-dependency-analyzer/blob/6fcbe9815398ef2530a085da901cc8e847a8daaa/src/main/java/org/apache/maven/shared/dependency/analyzer/asm/VisitClassException.java) defined in the package, should we use it ?